recipes/caf/all/conanfile.py

```diff
@@ -0,0 +1,140 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class CAFConan(ConanFile):
+    name = "caf"
+    description = "An open source implementation of the Actor Model in C++"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/actor-framework/actor-framework"
+    topics = "conan", "caf", "actor-framework", "actor-model", "pattern-matching", "actors"
+    license = "BSD-3-Clause, BSL-1.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "log_level": ["ERROR", "WARNING", "INFO", "DEBUG", "TRACE", "QUIET"],
+        "openssl": [True, False]
```

Comment:
```suggestion
        "with_openssl": [True, False]
```

Most recipes are using with_ let's try use the same.